HyperV/scripts/test_server.ps1

```diff
@@ -0,0 +1,48 @@
+

```

Comment:
looks like an older script not used in the CI.
It also has hard-coded pip version that is not ok. Do we need this?